<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Testisofta</string>
  </property>
  <widget class="QPushButton" name="car_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Car</string>
   </property>
  </widget>
  <widget class="QLabel" name="car_count">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>66</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="palaute">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>104</width>
     <height>93</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioYes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>109</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>yes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioNo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>109</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>no</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelReaction">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
